packages/vast-xml2js/src/helpers/xmlToJson.js

```diff
@@ -0,0 +1,66 @@
+const TYPE = {
+  CDATA: 'cdata',
+  DOCUMENT: 'document',
+  ELEMENT: 'element',
+  TEXT: 'text'
+};
+
+const nodeType = (node) => {
+  if (node.nodeType === 1) {
```

Comment:
Would be better to use constants instead. I.e. 
```if (node.nodeType === node.ELEMENT_NODE) { ... }```

Available constants: https://github.com/jindw/xmldom/blob/master/dom.js#L36

Reply:
we are only usnig xmldom on the frontend. Not sure those constants are on all browsers. Therfore I rather not to be honest

Reply:
now is too late but https://www.npmjs.com/package/xmldom